osquery/sql/sqlite_string.cpp

```diff
@@ -210,6 +210,90 @@ static void regexStringMatchFunc(sqlite3_context* context,
                       SQLITE_TRANSIENT);
 }
 
+/**
+ * @brief Concatenate strings ignoring nulls
+ */
+static void concatStringFunc(sqlite3_context* context,
```

Comment:
It would be nice if the two functions shared a common implementation with the separator set to `""` in this case.

Reply:
Yeah... I was thinking splitting them saved an `if` comparison, but that's probably premature optimization. I've merged them. I'm fumbling a bit with the c++